x-pack/plugins/apm/server/lib/services/get_service_transaction_groups/get_timeseries_data_for_transaction_groups.ts

```diff
@@ -84,7 +82,7 @@ export async function getTimeseriesDataForTransactionGroups({
         transaction_groups: {
           terms: {
             field: TRANSACTION_NAME,
-            size,
+            size: 500,
```

Comment:
why 500?

Reply:
I set the same value that was defined in the transaction groups query: https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/lib/services/get_service_transaction_groups/get_transaction_groups_for_page.ts#L97. If we might show 500 transaction groups, we must fetch the same 500 metrics for that, right?

Reply:
I'm not sure if we should? Why not only for the current page? In any case, you have a fixed set of transaction names, so you can just use transactionNames.length as size.